storage/storage_test.go

```diff
@@ -12,6 +12,8 @@ import (
 
 // a change
 
+//another change
```

Comment:
what's the deal here?
```suggestion
//another change is cool
```